src/app/views/functions/_function.html.erb

```diff
@@ -35,6 +35,20 @@
       <span class="icon"> <i class="fas fa-trash"></i> </span>
     <% end if can?(current_user, function, :destroy) %>
 
+    <% if save = Save.find_by(function_id: function, user_id: current_user) %>
```

Comment:
abstract this to a helper method in `SavesHelper` called `saved?` similar to the one in the `LikesHelper`